src/main/resources/application.yaml

```diff
@@ -71,6 +71,8 @@ reports:
   service-config:
     - service: cmc_claim_store
       display-name: CMC
+    - service: civil-service
+      display-name: CMC
```

Comment:
Are you sure display-name should be the same?

Reply:
raised https://github.com/hmcts/send-letter-service/pull/1788